Contest_Control_System_Requirements.md

```diff
@@ -80,7 +80,7 @@ document.
 
 ### Licensing
 
-The CCS must either be freely useable, or must be accompanied by a
+The CCS must either be freely usable, or must be accompanied by a
```

Comment:
I believe this is British vs American English, isn't it?

What are we using?

Reply:
FYI: we are using American English, see 8e433ffb0954fa42fbb606399d046e513304233f.